### DISCUSSION TITLE
spaceroom fix

### DISCUSSION THREAD

**Discussion by lelubunny** [Monika-After-Story/MonikaModDev#9429] [QA - tech support + general] [answered]
My MAS game won't open on my MacBook in the 12.09 update, everytime I open it says "While loading <'Image' u'mod_assets/location/Spaceroom V3.1/V3.1_rain.png'>:
  File "game/script-ch30.rpy", line 1541, in script call
  File "game/event-handler.rpy", line 3119, in script call
  File "game/script-story-events.rpy", line 1194, in script call
  File "game/script-story-events.rpy", line 1470, in script call
  File "game/script-story-events.rpy", line 1481, in script
IOError: Couldn't find file 'mod_assets/location/Spaceroom V3.1/V3.1_rain.png'."

**Comment by Wingdinggaster656** [comment, marked as answer]
Sounds like a submod problem. You are missing a file. Please reinstall.

**Comment by lelubunny** [comment]
On Wed, Aug 3, 2022 at 9:18 PM Spotted Dove ***@***.***>
wrote:

> Sounds like a submod problem. You are missing a file. Please reinstall.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Monika-After-Story/MonikaModDev/issues/9423#issuecomment-1204679478>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUT3JHQC7JIZU3JQUK5N7FDVXMR57ANCNFSM55P7XRUA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
thank you, i got it working:)
